src/azure-cli/Pipfile

```diff
@@ -0,0 +1,65 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+    azure-cli-telemetry = ">=1.0.0"
+    knack = ">=0.5.1"
+
+    azure-cli-acr = "*"
+    azure-cli-acs = "*"
+    azure-cli-advisor = "*"
+    azure-cli-ams = "*"
+    azure-cli-appservice  = "*"
+    azure-cli-backup = "*"
+    azure-cli-batch = "*"
+    azure-cli-batchai = "*"
+    azure-cli-billing = "*"
+    azure-cli-botservice = "*"
+    azure-cli-cdn = "*"
+    azure-cli-cloud = "*"
+    azure-cli-cognitiveservices = "*"
+    azure-cli-configure = "*"
+    azure-cli-consumption = "*"
+    azure-cli-container = "*"
+    azure-cli-core = "*"
+    azure-cli-cosmosdb = "*"
+    azure-cli-dla = "*"
+    azure-cli-dls = "*"
+    azure-cli-dms = "*"
+    azure-cli-eventgrid = "*"
+    azure-cli-eventhubs = "*"
+    azure-cli-extension = "*"
+    azure-cli-feedback = "*"
+    azure-cli-find = "*"
+    azure-cli-hdinsight = "*"
+    azure-cli-interactive = "*"
+    azure-cli-iot = "*"
+    azure-cli-iotcentral = "*"
+    azure-cli-keyvault = "*"
+    azure-cli-lab = "*"
+    azure-cli-maps = "*"
+    azure-cli-monitor = "*"
+    azure-cli-network = "*"
+    azure-cli-nspkg = "*"
+    azure-cli-policyinsights = "*"
+    azure-cli-profile = "*"
+    azure-cli-rdbms = "*"
+    azure-cli-redis = "*"
+    azure-cli-relay = "*"
+    azure-cli-reservations = "*"
+    azure-cli-resource = "*"
+    azure-cli-role = "*"
+    azure-cli-search = "*"
+    azure-cli-servicebus = "*"
+    azure-cli-servicefabric = "*"
+    azure-cli-signalr = "*"
+    azure-cli-sql = "*"
+    azure-cli-storage = "*"
+    azure-cli-vm = "*"
+
+[requires]
+python_version = "3.6"
```

Comment:
What about other Python versions?

Reply:
"requires" is added by the default template of pipenv but should be removed from the file if your app works with several Python